frontend/src/views/web/challenge/submission.html

```diff
@@ -12,78 +12,114 @@
                     </div>
                 </div>
                 <div class="ev-md-container ev-panel-title"><strong>Please fill the following fields to submit</strong></div>
-                <div class=" ev-card-body exist-team-card">
-                    <form ng-submit="challenge.makeSubmission()">
-                        <ul>
-                            <b>Select Phase:</b>
-                            <li ng-repeat="item in challenge.phases.results">
-                                <input type="radio" name="selectPhase" class="with-gap selectPhase" id="{{item.id}}" value="{{item.id}}" ng-model="challenge.phaseId">
-                                <label for="{{item.id}}"></label>
-                                <div class="show-member-title pointer"><strong class="text-med-black">Phase: </strong>{{item.name}}
-                                    <br>
-                                    <strong class="text-med-black">End Date: </strong>{{item.end_date | date:'medium'}}</div>
-                                <div class="clearfix"></div>
-                            </li>
+                <div class="col s12 m7">
+                    <div class="ev-card-body exist-team-card">
+                        <form ng-submit="challenge.makeSubmission()">
+                            <ul>
+                                <b>Select Phase:</b>
+                                <li ng-repeat="item in challenge.phases.results">
+                                    <input type="radio" name="selectPhase" class="with-gap selectPhase" id="{{item.id}}" value="{{item.id}}" ng-model="challenge.phaseId" ng-click="challenge.showRemainingSubmissions()">
+                                    <label for="{{item.id}}"></label>
+                                    <div class="show-member-title pointer"><strong class="text-med-black">Phase: </strong>{{item.name}}
+                                        <br>
+                                        <strong class="text-med-black">End Date: </strong>{{item.end_date | date:'medium'}}</div>
+                                    <div class="clearfix"></div>
+                                </li>
 
-                            <div class="row">
-                                <div class="file-field input-field col s6">
-                                    <div class="waves-effect waves-dark btn ev-btn-dark w-300 fs-14">
-                                        <span>Upload File</span>
-                                        <input type="file" ngf-select ng-model="challenge.input_file" name="challenge.input_file" accept=".json">
-                                    </div>
-                                    <div class="file-path-wrapper">
-                                        <input class="file-path validate" type="text">
+                                <div class="row">
```

Comment:
Why row here if no col presents? Main thing about grid is to use row whenever we have to group columns. And all col should be present in a row.